### PR TITLE
Dark Mode - NxStatusIndicator - RSC-1527

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1527

### CODE REVIEW

File: lib/src/base-styles/_nx-colors.scss [hunk 1]
```diff
@@ -196,6 +196,31 @@
...
+  --nx-color-status-indicator-negative-background: var(--nx-swatch-grey-95);
+  --nx-color-status-indicator-negative-border: var(--nx-swatch-grey-30);
+  --nx-color-status-indicator-negative-text: var(--nx-swatch-grey-10);
+  --nx-color-status-indicator-negative-icon-background: var(--nx-swatch-grey-60);
+  --nx-color-status-indicator-negative-icon-border: var(--nx-swatch-grey-20);
+
+  --nx-color-status-indicator-intermediate-background: var(--nx-swatch-blue-95);
+  --nx-color-status-indicator-intermediate-border: var(--nx-swatch-blue-30);
+  --nx-color-status-indicator-intermediate-text: var(--nx-swatch-blue-10);
+  --nx-color-status-indicator-intermediate-icon-background: var(--nx-swatch-blue-30);
```
Comment (cindyko226) [inline lib/src/base-styles/_nx-colors.scss:209]: In sketch, the color of intermediate icon background is set to `nx-blue-60` in light mode

Comment (avnvbhatta) [reply lib/src/base-styles/_nx-colors.scss]: > In sketch, the color of intermediate icon background is set to `nx-blue-60` in light mode

Good catch! I ran the light mode visual test and didn't see any new snapshots, so I thought I all of my color variables were correct. But there's actually a bug in `main` currently. If you go to gallery.sonatype.dev it actually has the wrong background color (it's blue-30). 😄 

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/1069/commits/300089e2f6d9572ae9faeb9766ee39f9e65c7b15

Comment (cindyko226) [reply lib/src/base-styles/_nx-colors.scss]: Yes the color in main now is incorrect. This should fix that once merged.

**Review by cindyko226 — APPROVED** [review]
LGTM

**Pull request merged**